nemo/collections/tts/modules/audio_codec_modules.py

```diff
@@ -698,3 +716,429 @@ def decode(self, indices: torch.Tensor, input_len: torch.Tensor) -> torch.Tensor
         dequantized = torch.cat(dequantized, dim=1)
 
         return dequantized
+
+
+class ResidualBlock(NeuralModule):
+    """
+    The residual block structure defined by the HiFi-GAN V1 and V2 configurations.
+    """
+
+    def __init__(
+        self,
+        in_channels: int,
```

Comment:
minor: Are you following any published model here? Otherwise rename `in_channels` to `channels`? 

Doc string missing for args.

Reply:
Changed to `channels` and added doc string.